server/package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "npm run build:live --runInBand",
     "build:live": "tsnd --watch 'src/**/*.ts' --respawn --transpileOnly --inspect -- src/index.ts",
-    "prisma": "prisma"
+    "prisma": "prisma",
+    "prisma:seed": "prisma reset && tsnd prisma/seed.ts"
```

Comment:
I don't see why we want to use `prisma reset` command here, from the docs [prisma reset] (https://www.prisma.io/docs/prisma-cli-and-configuration/cli-command-reference/prisma-reset-ycv3/) it deletes the entire DB.

Reply:
Precisely, `prisma reset` deletes all the data inside db, not db itself. That was intended to avoid collision with existing data  - if data (user e-mail) already exists in db, attempt to `prisma.createNewbie()` will fail, as such user already exists.  My intention was to cleanup db and insert sample data from seed file. When this script became useful?

1. Every time when new environment is configured and deployed for the first time, for instance:
- dev.buddyapp.buildit.digital
- staging.buddyapp.buildit.digital
- prod.buddyapp.buildit.digital

_(Of course if we gonna create so many envs)_ ☝️

2. When we want to cleanup db data for some reasons on selected environment.

However, I could remove `prisma reset` from package.json's script `"prisma reset && tsnd prisma/seed.ts"` and change current implementation to upsert data, instead of creating new, but in such situation I can't see why we want to do that in the first place? If we have db with some data, what's the point to add to db those specific sample users and tasks defined in server/prisma/seedData.ts ? 🤔

Like I explained in 1) and 2) if you run this script for the first time when new environment is setup, you don't need to add sample data manually. Hope it makes sense for you.

Reply:
We have just discussed with @lucianojss problem with reseting DB before seeding it with seedfile - Luciano would like to remove it and we've got to the point, where I could use your help with this issue and decide. Luciano is not convinced so I will leave here two sources that I based on, also read my explanation  in the comment above☝️Hope you will find it interesting @Tonik1204 @duiliopastorelli 

1. **[Prisma Docs (prisma seed)](https://www.prisma.io/docs/prisma-cli-and-configuration/cli-command-reference/prisma-seed-xcv8/)**
On official documentation there's a flag that can reset data, before seeding.
![image](https://user-images.githubusercontent.com/14184100/73061417-6f374400-3e9a-11ea-861f-5f930dd15361.png)

2. **[How to reset and seed a prisma.io database](https://www.danilucaci.com/blog/reset-and-seed-prisma-database)**
A blog article about seeding prisma DB

![How to reset and seed a prisma io database 2020-01-24 11-15-29](https://user-images.githubusercontent.com/14184100/73061633-ea005f00-3e9a-11ea-9529-5d0b7125f89d.jpg)

Reply:
@michaelmoney @lucianojss  Both actions seem reasonable to me. In my opinion there should be two commands: one for situation when we need to reset data, and other just to upsert existing data

Reply:
@Tonik1204 @lucianojss Seems reasonable. I've added two scripts in this commit 4526b22754fbb14d7ff96005cc6f948932a63b68, slightly refactored to take advantage of `seed` field inside  `prisma.yml`.